zeebe/exporters/camunda-exporter/src/test/java/io/camunda/CamundaExporterHandlerIT.java

```diff
@@ -377,6 +378,15 @@ void shouldExportUsingOperationFromVariableDocumentHandler(
     testForOperationHandlers(handler, config, clientAdapter);
   }
 
+  @TestTemplate
+  void shouldExportUsingMappingCreatedHandler(
+      final ExporterConfiguration config, final SearchClientAdapter clientAdapter)
+      throws IOException {
+    final var handler = getHandler(config, MappingCreatedHandler.class);
+    basicAssertWhereHandlerCreatesDefaultEntity(
+        handler, config, clientAdapter, defaultRecordGenerator(handler));
```

Comment:
💭 I believe if you set specific intents in the handler then there is a different method you need to use here to generate the record. I believe it is [this method](https://github.com/camunda/camunda/blob/main/zeebe/protocol-test-util/src/main/java/io/camunda/zeebe/test/broker/protocol/ProtocolFactory.java#L186).

The `defaultRecordGenerator` method here uses the [base method](https://github.com/camunda/camunda/blob/main/zeebe/protocol-test-util/src/main/java/io/camunda/zeebe/test/broker/protocol/ProtocolFactory.java#L168) which selects a random intent (which is likely not to be one that your handler is configured to support)